sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -506,16 +504,16 @@ static inline void _nib_dst_remove(_nib_offl_entry_t *nib_offl, uint8_t mode)
  *
  * @note    Only available if @ref GNRC_IPV6_NIB_CONF_DC.
  *
- * @return  A new or existing destination cache entry with
- *          _nib_onl_entry_t::ipv6 of _nib_dr_entry_t::next_hop set to
- *          @p next_hop.
+ * @return  A new or existing off-link entry with _nib_dst_entry_t::pfx set to
+ *          @p pfx.
  * @return  NULL, if no space is left.
  */
 static inline _nib_offl_entry_t *_nib_dc_add(const ipv6_addr_t *next_hop,
                                              unsigned iface,
                                              const ipv6_addr_t *dst)
 {
-    return _nib_dst_add(next_hop, iface, dst, IPV6_ADDR_BIT_LEN, _DC);
+    assert(next_hop != NULL);
```

Comment:
But it's checked in `_nib_offl_alloc()` (which is called by `_nib_offl_add()`)...